.github/workflows/build.yml

```diff
@@ -373,12 +392,7 @@ jobs:
       - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
       - uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4.0.1
         with:
-          # pinning this to 1.20.5 because this issue in go-testcontainers occurs
-          # in 1.20.6 with the error "http: invalid Host header, host port waiting failed"
-          # https://github.com/testcontainers/testcontainers-go/issues/1359
-          # remove setting this when the above issue is fixed so that the job will
-          # just get the go version from .go-version.
-          go-version: 1.20.5
+          go-version: ${{ needs.get-go-version.outputs.go-version }}
```

Comment:
I think we need a `needs` added above (did we miss it on the main PR?)

Reply:
Yes, it is missing in the main PR. I will fix.